__tests__/app.test.js

```diff
@@ -55,6 +55,7 @@ describe('GET /api/articles:article_id', () => {
             expect(article.created_at).toBe("2020-07-09T20:11:00.000Z")
             expect(article.votes).toBe(100)
             expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+            expect(article.comment_count).toBe("11")
```

Comment:
This should have been it's own test, rather than tacking it onto an existing test (which is bad practice)